### ISSUE TITLE
Track random seeds

### ISSUE THREAD

**Issue by jvdzwaan** [recipy/recipy#105]
Related to #48; possible solution is #104 (although it is not effortless)

**Comment by robintw** [comment]
Yes, this is definitely something we should do - unfortunately I can't see an effortless way at the moment. We'd need to wait until modules like `random` or `numpy` are imported, and then grab the seeds...but of course the seeds may be set manually sometime after they're imported (tho I guess that will be in the code that we grab too).

We could potentially put some sort of callback/hook into our 'module loading hook' that will grab important information about the module that has been imported. In this case it could include the random number seed (eg. `np.random.seed`), but could also include versions of the modules/dependencies - also fixing #87

**Comment by robintw** [comment]
See http://stackoverflow.com/questions/32172054/how-can-i-retrieve-the-current-seed-of-numpys-random-number-generator for info
